dipy/reconst/tests/test_dti.py

```diff
@@ -185,6 +185,11 @@ def test_fa_of_zero():
     fa = fractional_anisotropy(evals)
     assert_array_equal(fa, 0)
 
+def test_ga_of_zero():
+    evals = np.zeros((4, 3))
```

Comment:
Just to be sure I got it: these are 4 voxels that each have the same evals of [0, 0, 0]?